### ISSUE TITLE
Trezor (classic): hid dialog exception

### ISSUE THREAD

**Issue by molecular** [akhavr/electrum-dash#20]
This happens in the wallet setup wizard when I try to setup a new trezor-based wallet after entering the passphrase.

```
Traceback (most recent call last):
  File "./electrum-dash", line 382, in <module>
    d.init_gui(config, plugins)
  File "/home/nick/bitcoin/electrum-dash/lib/daemon.py", line 305, in init_gui
    self.gui.main()
  File "/home/nick/bitcoin/electrum-dash/gui/qt/__init__.py", line 252, in main
    if not self.start_new_window(path, self.config.get('url')):
  File "/home/nick/bitcoin/electrum-dash/gui/qt/__init__.py", line 202, in start_new_window
    wallet = wizard.run_and_get_wallet()
  File "/home/nick/bitcoin/electrum-dash/gui/qt/installwizard.py", line 274, in run_and_get_wallet
    self.run(action)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 60, in run
    f(*args)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 86, in new
    self.choice_dialog(title=title, message=message, choices=choices, run_next=self.on_wallet_type)
  File "/home/nick/bitcoin/electrum-dash/gui/qt/installwizard.py", line 91, in func_wrapper
    run_next(*out)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 96, in on_wallet_type
    self.run(action)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 60, in run
    f(*args)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 128, in choose_keystore
    self.choice_dialog(title=title, message=message, choices=choices, run_next=self.run)
  File "/home/nick/bitcoin/electrum-dash/gui/qt/installwizard.py", line 91, in func_wrapper
    run_next(*out)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 60, in run
    f(*args)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 206, in choose_hw_device
    self.choice_dialog(title=title, message=msg, choices=choices, run_next=self.on_device)
  File "/home/nick/bitcoin/electrum-dash/gui/qt/installwizard.py", line 91, in func_wrapper
    run_next(*out)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 222, in on_device
    self.derivation_dialog(f)
  File "/home/nick/bitcoin/electrum-dash/lib/base_wizard.py", line 225, in derivation_dialog
    default = bip44_derivation(0, False)
TypeError: bip44_derivation() takes 1 positional argument but 2 were given
```
Happens both then using master branch and also Electrum-DASH-3.0.6.tar.gz

**Comment by molecular** [comment]
this fixes it:

```
diff --git a/lib/base_wizard.py b/lib/base_wizard.py
index d53f1c30..21ef5117 100644
--- a/lib/base_wizard.py
+++ b/lib/base_wizard.py
@@ -222,7 +222,7 @@ class BaseWizard(object):
             self.derivation_dialog(f)
 
     def derivation_dialog(self, f):
-        default = bip44_derivation(0, False)
+        default = bip44_derivation(0)
         message = '\n'.join([
             _('Enter your wallet derivation here.'),
             _('If you are not sure what this is, leave this field unchanged.')
```

**Comment by akhavr** [comment]
Which version of `trezor` package you've used?

**Comment by akhavr** [comment]
Checked.  Your fix would be incorporated in the next release.

**Comment by akhavr** [comment]
Fixed in 3.0.6.3 release

**Comment by molecular** [comment]
thank you!
